src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENTAL_COST = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const LONG_TERM_DURATION = 7;
+  const SHORT_TERM_DURATION = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM_DURATION) {
+    return DAY_RENTAL_COST * days - LONG_TERM_DISCOUNT;
```

Comment:
The discount is subtracted directly from the total cost. Ensure that this is the intended behavior. If the discount should be applied per day, adjust the calculation accordingly.